doc/Development_Documentation/03_Documents/03_Navigation.md

```diff
@@ -287,85 +285,78 @@ use Twig\TwigFunction;
 {
-        /**
-         * @var Navigation
-         */
-        protected $navigationHelper;
-    
-        /**
-         * @var NewsLinkGenerator
-         */
-        protected $newsLinkGenerator;
-    
-        /**
-         * @param Navigation $navigationHelper
-         */
-        public function __construct(Navigation $navigationHelper, NewsLinkGenerator $newsLinkGenerator)
-        {
-            $this->navigationHelper = $navigationHelper;
-            $this->$newsLinkGenerator = $newsLinkGenerator;
-        }
-        /**
-         * @return array|TwigFunction[]
-         */
-        public function getFunctions()
-        {
-            return [
-                new TwigFunction('app_navigation_news_links', [$this, 'getDataLinks'])
-            ];
-        }
-        
-            /**
-             * @param Document $document
-             * @param Document $startNode
-             *
-             * @return \Pimcore\Navigation\Container
-             * @throws \Exception
-             */
-            public function getNewsLinks(Document $document, Document $startNode)
-            {
-                $navigation = $this->navigationHelper->build([
-                    'active' => $document,
-                    'root' => $startNode,
-                    'pageCallback' => function($page, $document) {
-                        /** @var \Pimcore\Model\Document $document */
-                        /** @var \Pimcore\Navigation\Page\Document $page */
-                        if($document->getProperty("templateType") == "news") {
-                            $list = new \Pimcore\Model\DataObject\News\Listing;
-                            $list->load();
-                            foreach($list as $news) {
-                                $detailLink = $this->newsLinkGenerator->generate($news, ['document' => $document]);
-                                $uri = new \Pimcore\Navigation\Page\Document([
-                                    "label" => $news->getTitle(),
-                                    "id" => "object-" . $news->getId(),
-                                    "uri" => $detailLink,
-                                ]);
-                                $page->addPage($uri);
-                            }
-                        }
+    protected Navigation $navigationHelper;
+    protected NewsLinkGenerator $newsLinkGenerator;
+
+    /**
+     * @param Navigation $navigationHelper
+     * @param NewsLinkGenerator $newsLinkGenerator
+     */
+    public function __construct(Navigation $navigationHelper, NewsLinkGenerator $newsLinkGenerator)
+    {
+        $this->navigationHelper = $navigationHelper;
+        $this->$newsLinkGenerator = $newsLinkGenerator;
+    }
+
+    /**
+     * @return TwigFunction[]
+     */
+    public function getFunctions()
+    {
+        return [
+            new TwigFunction('app_navigation_news_links', [$this, 'getDataLinks'])
+        ];
+    }
+
+    /**
+     * @param Document $document
```

Comment:
we don't add phpdocs, if there are typehints present. please remove it.

Reply:
This one was missing from the ones added already.